entry_types/scrolled/package/src/frontend/Backdrop.js

```diff
@@ -146,13 +157,20 @@ function BackgroundVideo(props) {
   }, [playerActions])
 
   return (
-    <VideoPlayer isPrepared={isPrepared}
-                 playerState={playerState}
-                 playerActions={playerActions}
-                 id={props.video}
-                 textTracksDisabled={true}
-                 fit="cover"
-                 loop={true}
-                 playsInline={true} />
+    <>
+      <VideoPlayer isPrepared={isPrepared}
+                   playerState={playerState}
+                   playerActions={playerActions}
+                   id={video.permaId}
+                   textTracksDisabled={true}
+                   fit="cover"
+                   loop={true}
+                   playsInline={true} />
+      <MotifArea key={video.permaId}
+                 onUpdate={onMotifAreaUpdate}
+                 file={video}
+                 containerWidth={containerDimension.width}
+                 containerHeight={containerDimension.height}/>
+    </>
```

Comment:
**nitpick:** This file is becoming long. We could move components to individual files. It is okay if this happens in a different PR than this one

Reply:
I agree. Since I already have two more PRs lined up that build upon the changes here, I'll try to keep this in mind for later.

Reply:
Better than trying to keep it in mind would be to add a checklist item to any Redmine issue that is suitable according to you